src/herder/TxSetFrame.cpp

```diff
@@ -521,6 +521,13 @@ TxSetFrame::checkValid(Application& app, uint64_t lowerBoundCloseTimeOffset,
         return false;
     }
 
+    if (!std::is_sorted(mTxs.begin(), mTxs.end(), &TxSetUtils::hashTxSorter))
```

Comment:
I think we should use TxSetUtils::isValidHashOrder

Reply:
As far as I can tell, this function no longer exists.

Reply:
Yeah, I removed it because it has been used in a single place before and the comparator is now exposed... I guess we could bring it back, given that we need this twice again now.

Reply:
ah sorry, was looking at the wrong branch. All good then!